modules/spo-memory-optimzation.adoc

```diff
@@ -21,7 +21,7 @@ SPO memory optimization is not enabled by default.
 +
 [source,terminal]
 ----
-$ oc -n openshift-security-profiles patch spod spod --type=merge -p '{"spec":{"enableMemoryOptimization":true}}'
+$ oc -n openshift-security-profiles patch daemonset spod --type=merge -p '{"spec":{"enableMemoryOptimization":true}}'
```

Comment:
This is actually a valid command, but there might be another way to update the limits using the daemonset.

Reply:
Thanks for catching this - looks like it's written the same upstream: https://github.com/kubernetes-sigs/security-profiles-operator/blob/main/installation-usage.md#enable-memory-optimization-in-spod

`kubectl -n security-profiles-operator patch spod spod --type=merge -p '{"spec":{"enableMemoryOptimization":true}}'`

Reply:
@xiaojiey  can you review when you get a chance? Thanks!

Reply:
```
There is the same issue for downstream build:
$ oc -n openshift-security-profiles patch daemonset spod --type=merge -p '{"spec":{"enableMemoryOptimization":true}}'
Warning: unknown field "spec.enableMemoryOptimization"
daemonset.apps/spod patched (no change)
$ oc -n openshift-security-profiles get spod/spod -o=jsonpath={.spec.enableMemoryOptimization}
$
```